### PR TITLE
Depends: Update FreeType package (CVE-2020-15999)

### PR DESCRIPTION
**Pull request by mammix2**
Update FreeType package to v2.10.4 in Depends to help mitigate against recent 0 day flaw found. 
See CVE-2020-15999

### CODE REVIEW

File: depends/packages/freetype.mk [hunk 1]
```diff
@@ -1,8 +1,8 @@
 package=freetype
-$(package)_version=2.7.1
-$(package)_download_path=https://download.savannah.gnu.org/releases/$(package)
-$(package)_file_name=$(package)-$($(package)_version).tar.bz2
-$(package)_sha256_hash=3a3bb2c4e15ffb433f2032f50a5b5a92558206822e22bfe8cbe339af4aa82f88
+$(package)_version=2.10.4
+$(package)_download_path=http://download.savannah.gnu.org/releases/$(package)
```
Comment (luke-jr) [inline depends/packages/freetype.mk:3]: No opinion on whether to use http vs https

File: depends/packages/freetype.mk [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+$(package)_file_name=$(package)-$($(package)_version).tar.xz
+$(package)_sha256_hash=86a854d8905b19698bbc8f23b860bc104246ce4854dcea8e3b0fb21284f75784
```
Comment (luke-jr) [inline depends/packages/freetype.mk:5]: This matches what I've had on my system since 2020-10-20

**Review by luke-jr — APPROVED** [review]
utACK

File: depends/packages/freetype.mk (hunk 1 above)
Comment (MarcoFalke) [reply depends/packages/freetype.mk:3]: Indeed. Also, please update the docs as done in commit 83582dcf23c23326fcb648b436d51469e66db5d5

Comment (mammix2) [reply depends/packages/freetype.mk:3]: No probs, i'll test with https and update the docs too.
Closing

**Pull request closed without merging**